### PR TITLE
Styleguide Updates

### PR DESCRIPTION
**Pull request by smbriones**
screen shot 2017-03-07 at 5 29 59 pm

### CODE REVIEW

**Comment by cheshire137** [comment]
Oh thank you! I wondered after I looked at the styleguide running locally if it would be helpful to put the hex values in there, but even better is the actual variable names, so we don't go encouraging people to copy-paste the hex values!

File: app/assets/stylesheets/structure.scss [hunk 1]
```diff
@@ -1,10 +1,55 @@
+*, *:before, *:after {
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
```
Comment (cheshire137) [inline app/assets/stylesheets/structure.scss]: Looks like there's a [Compass mix-in](http://compass-style.org/examples/compass/css3/box_sizing/) for box-sizing, if you want to combine lines 2-4.

File: app/assets/stylesheets/structure.scss [hunk 2]
```diff
@@ -1,10 +1,55 @@
...
+@media screen and (min-width: 800px) {
   .container {
-    margin: 0 auto;
     max-width: 1020px;
   }
+
+  .flex-grid {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
```
Comment (cheshire137) [inline app/assets/stylesheets/structure.scss]: You can use [Compass' display-flex](http://compass-style.org/reference/compass/css3/flexbox/#mixin-display-flex) to combine lines 30-32.

File: app/assets/stylesheets/structure.scss [hunk 3]
```diff
@@ -1,10 +1,55 @@
...
+    .col { flex: 1; }
+  }
+
+  .flex-grid-thirds {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-pack: justify;
+    -ms-flex-pack: justify;
+    justify-content: space-between;
```
Comment (cheshire137) [inline app/assets/stylesheets/structure.scss]: Compass will let you combine lines 37-39 and then 40-42 using mix-ins.

File: app/assets/stylesheets/structure.scss [hunk 4]
```diff
@@ -1,10 +1,55 @@
...
+    .col { width: 32%; }
+  }
+
+  .flex-grid-half {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-pack: justify;
+    -ms-flex-pack: justify;
+    justify-content: space-between;
```
Comment (cheshire137) [inline app/assets/stylesheets/structure.scss]: Think of the Compass savings here! 🤑

File: app/assets/stylesheets/styleguide.scss [hunk 5]
```diff
@@ -1,38 +1,50 @@
...
+  flex-direction: row;
   margin-bottom: 2em;
-  overflow: hidden;
   li {
-    display: inline-block;
-    float: left;
-    height: 50px;
+    display: flex;
+    border-radius: 3px;
+    flex-grow: 1;
```
Comment (cheshire137) [inline app/assets/stylesheets/styleguide.scss]: Another Compass mix-in for you: http://compass-style.org/reference/compass/css3/flexbox/#mixin-flex-grow

File: app/assets/stylesheets/styleguide.scss [hunk 6]
```diff
@@ -1,38 +1,50 @@
...
-    &.soft-header {
-      background: $soft-header;
+  }
+}
+
+// Grid
+.flexbox-grid-demo {
+  .flex-grid, .flex-grid-thirds, .flex-grid-half {
+    .col {
+      align-items: center;
```
Comment (cheshire137) [inline app/assets/stylesheets/styleguide.scss]: Compass again: http://compass-style.org/reference/compass/css3/flexbox/#mixin-align-items

**Comment by smbriones** [comment]
@cheshire137 Yes, love Compass! Just wasn't sure if it was added yet.

**Comment by cheshire137** [comment]
Oh good! I ended up adding it when the [gradient generator](http://www.colorzilla.com/gradient-editor/) I used for replicating your mockup's header gradient had a SCSS output option and it required Compass. See application.scss for where Compass is getting included.

**Comment by smbriones** [comment]
Beautiful. Thanks! I'll update to use Compass.

**Pull request merged**